test/Performance/.gitignore

```diff
@@ -0,0 +1,5 @@
+/ext/
+/*.html
+/*.csv
+/*.xml
```

Comment:
excluding .xml files looks a bit aggressive, since there are .xml files in the repo... maybe better to trust that renaming *xml to *xml.bak will prevent git to pick those files up?